src/main/java/array/Exercise05.java

```diff
@@ -0,0 +1,23 @@
+package array;
+
+/**
+ * Exercise05: A whole number of n elements values from 1 to n, removing any value and then randomly arranged.
+ * Write a program to receive the integer array of the above process and search for the number that has been removed.
+ */
+public class Exercise05 {
+    public int findMissingNumber(int[] arr) {
+        final int n = arr.length + 1;
+
+        int xorAll = 0;
+        for (int i = 0; i < n; i++) {
+            xorAll ^= i;
+        }
+
+        int xorArr = 0;
+        for (int num : arr) {
+            xorArr ^= num;
+        }
+
+        return xorAll ^ xorArr;
```

Comment:
This approach works fine (and could even be considered best practice), but since the ^= operator is rarely used, the simplest solution is just sum(1..n) - sum(arr).